wrapper/src/libvmaf.h

```diff
@@ -19,6 +19,8 @@
 #ifndef LIBVMAF_H_
 #define LIBVMAF_H_
 
+#define MAX_NUM_VMAF_MODELS 5 // there can be MAX_NUM_VMAF_MODELS - 1 additional models at most
```

Comment:
Why is there a maximum number of models?

Reply:
This has been done to simplify the generation of the additional models. Will bump it up to 20.